src/components/TheSidebar.vue

```diff
@@ -9,11 +9,11 @@
       </p>
     </div>
 
-    <SmallBoard :listitems="recent">
+    <SmallBoard :listitems="recent" :sidebar="true">
```

Comment:
You can just use `<SmallBoard :listitems="recent" sidebar>`. But as it doesn't matter, you don't need to fix it.